app/assets/javascripts/views/project_view.js

```diff
@@ -1,17 +1,23 @@
 var StoryView = require('./story_view');
 var IterationView = require('./iteration_view');
+var ColumnView = require('./column_view');
+var ColumnVisibilityButtonView = require('./column_visibility_button_view');
 
 module.exports = Backbone.View.extend({
+  template: require('templates/project_view.ejs'),
   columns: {},
 
   initialize: function() {
 
-    _.bindAll(this, 'addStory', 'addAll', 'render');
+    _.bindAll(this, 'addStory', 'addAll', 'render', 'prepareColumns');
 
     this.model.stories.on('add', this.addStory);
     this.model.stories.on('reset', this.addAll);
     this.model.stories.on('all', this.render);
     this.model.on('change:userVelocity', this.addAll);
+    this.model.on('change:inverse_flow', this.prepareColumns);
```

Comment:
You'd rather use `this.listenTo(this.model, ''change:inverse_flow'', this.prepareColumns);` instead just `model.on`.